### PR TITLE
Add feature: Print breakpoint table

### PR DESCRIPTION
**Pull request by yihuajack**
Take example5.evcd, breakpoints=[9,11,17] as example, the table is:

```
time    9       11      17
scope   testbench
scope   pe8
req_I   128     16      4
req_O   zzzzzzzz        zzzzzzzz        zzzzzzzz
en_I    1       1       1
en_O    z       z       z
scope   pe4
req_I   8       4       2
req_O   zzzz    zzzz    zzzz
en_I    1       1       1
en_O    z       z       z
```

### CODE REVIEW

**Comment by yihuajack** [comment]
The column width of tables can be fine-adjusted, but to achieve it, sootty needs to recursively see all the wires twice, so performance is lost.

File: sootty/storage/wiregroup.py [hunk 1]
```diff
@@ -40,9 +38,27 @@ def find(self, name: str):
 
     def get_names(self):
```
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:39]: This change to how wire names are computed should be a new PR, assuming it's separate from the breakpoint logic.

File: sootty/storage/wiregroup.py [hunk 2]
```diff
@@ -40,9 +38,27 @@ def find(self, name: str):
...
+                    names[self.name].append(wire.name)
+            for group in self.groups:
+                names[group.name] = group.get_names()
+        else:
+            names = list()
+            for wire in self.wires:
+                names.append(wire.name)
         return names
+    
+    def get_wires(self):
```
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:55]: Add docstring comment for what the function does.

**Review by Ben1152000 — CHANGES_REQUESTED** [review]

**Comment by Ben1152000** [comment]
Have you tested if this works when the waveform is being displayed? The svg is written to stdout and piped to the display program, so if the breakpoints are also printed to stdout that could possibly break the image.

File: sootty/storage/wiregroup.py (hunk 1 above)
Comment (yihuajack) [reply sootty/storage/wiregroup.py:39]: now resolved in: https://github.com/Ben1152000/sootty/pull/64

**Comment by yihuajack** [comment]
> Have you tested if this works when the waveform is being displayed? The svg is written to stdout and piped to the display program, so if the breakpoints are also printed to stdout that could possibly break the image.

Yes, I've tested that this works with or without `-o`, but I'm not quite familiar with `subprocess`. What if `subprocess.run` is used instead of `subprocess.Popen()`?

**Review by Ben1152000 — APPROVED** [review]

File: sootty/storage/wiregroup.py [hunk 3]
```diff
@@ -39,10 +37,29 @@ def find(self, name: str):
         raise SoottyError(f"Wire '{name}' does not exist.")
 
     def get_names(self):
-        """Returns list of all wire names."""
-        names = set()
-        for wire in self.wires:
-            names.add(wire.name)
-        for group in self.groups:
-            names.update(group.get_names())
+        """Returns a dictionary of all wire names of this wiregroup or a list if this wiregroup is the innermost one."""
```
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:40]: Can you remove the duplicate changes from PR #64? I approved so once you do that you can merge it in.

**Review by Ben1152000 — APPROVED** [review]

File: sootty/storage/wiregroup.py (hunk 3 above)
Comment (yihuajack) [reply sootty/storage/wiregroup.py:40]: Removed

**Pull request merged**